### ISSUE TITLE
NetSparker scan import issue

### ISSUE THREAD

**Issue by MotiejusK** [DefectDojo/django-DefectDojo#9816]
**Bug description**
Issue is either in demo environment (https://demo.defectdojo.org/) either in latest on premise installation .
Tried either sample scan file from https://github.com/DefectDojo/sample-scan-files/blob/master/netsparker/Netsparker_v0.1.json either own file.

**Steps to reproduce**
1. Go to product
2. Go to engagement
3. click import scan
4. pick netsparker
5. attach file.
6.submit

**Expected behavior**
Scan results imported

**Deployment method** *(select with an `X`)*
- [X] Docker Compose
- [ ] Kubernetes
- [ ] GoDojo

**Sample scan files**
https://github.com/DefectDojo/sample-scan-files/blob/master/netsparker/Netsparker_v0.1.json

**Screenshots**
![image](https://github.com/DefectDojo/django-DefectDojo/assets/11046392/f27650da-7af1-462a-8adb-28be09f69362)

**Comment by manuel-sommer** [comment]
Maybe this relates to https://github.com/DefectDojo/django-DefectDojo/issues/9712

**Comment by manuel-sommer** [comment]
See PR @MotiejusK

**Comment by manuel-sommer** [comment]
Hi @MotiejusK , this can be closed

**Comment by mtesauro** [comment]
@MotiejusK Since that PR was merged, it will be in Monday's release of 2.33.0 - Monday evening (US Central time) the demo site will be upgraded to 2.33.0. You can either upgrade your installation or try this out in the public demo site once that upgrade happens.

Historically, we've closed issues once the PR is merged that addresses them since those PRs include unit tests to cover the issue they address. If the PR fails to address the issue, another issue can be opened or this issue can be re-opened as well.

I would actually appreciate you re-opening this if the issues isn't resolved in Monday's release of 2.33.0. :+1:

**Comment by MotiejusK** [comment]
In generally fix is working, but got into other issue :

An exception error occurred during the report import: 'NoneType' object has no attribute 'encode'. System crashes to import at some line..

I would note that Nerspaker json file in sample is quite old. Old one imports well, but current version Netsparker files rises error.

**Comment by manuel-sommer** [comment]
@MotiejusK can you please test #10311 ?

**Comment by MotiejusK** [comment]
@manuel-sommer  where ?

**Comment by manuel-sommer** [comment]
@MotiejusK see above https://github.com/DefectDojo/django-DefectDojo/pull/10312
